source/transifex-master/en.json

```diff
@@ -63,6 +65,7 @@
   "wordpress": "WordPress",
   "tor": "Tor",
   "integrations-and-plugins": "Integrations and plugins",
+  "integrations-and-plugins-api-text": "And there is also our API, which offers full programmatic access to your BTCPay Server instance.",
```

Comment:
This can be deleted IMO.

It doesn't make quite total sense to have this sentence talking about a single item from the above integration icons and not the others.
It also feels redundant now that we have an API link in the header, an API logo with link, and an API simple sentence in the "features" secondary nav item. It's also the exact same sentence than in the secondary nav bar mentioned just before.
Also, one less sentence to translate.

Reply:
You are right, this is obsolete.